test/e2e_node/system/types_unix.go

```diff
@@ -63,7 +63,7 @@ var DefaultSysSpec = SysSpec{
 	RuntimeSpec: RuntimeSpec{
 		DockerSpec: &DockerSpec{
 			Version:     []string{`1\.1[1-3]\..*`, `17\.03\..*`}, // Requires [1.11, 17.03]
-			GraphDriver: []string{"aufs", "overlay", "overlay2", "devicemapper"},
+			GraphDriver: []string{"aufs", "overlay", "overlay2", "devicemapper", "zfs"},
```

Comment:
In general this seems fine. My only question is that since this system spec is also used in e2e_node_suite_test.go, whether this implies any particular level of support for docker + zfs.

cc @timothysc @derekwaynecarr

Reply:
my only confirmation that people are actually using ZFS + Docker + k8s is the issue report that triggered this PR:
https://github.com/kubernetes/kubeadm/issues/930#issuecomment-398119024

and also this blog post:
https://hackernoon.com/starting-small-with-kubernetes-and-kubeadm-500-containers-6c76454e6e12

so in a way people are already using it, but not sure how a particular level of support can be defined for a Docker graph driver.